modules/siddhi-extensions/map/src/test/java/org/wso2/siddhi/extension/map/ToXMLFunctionExtensionTestCase.java

```diff
@@ -329,4 +330,161 @@ public void receive(Event[] inEvents) {
         Assert.assertTrue(eventArrived);
         executionPlanRuntime.shutdown();
     }
+
+    @Test(expected = ExecutionPlanValidationException.class)
+    public void testToXMLFunctionExtension5() throws InterruptedException {
+        log.info("ToXMLFunctionExtension TestCase with test attributeExpressionExecutors length");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        siddhiManager.setExtension("str:concat", ConcatFunctionExtension.class);
+
+        String inStreamDefinition = "\ndefine stream inputStream (symbol string, price long, volume long);";
+        String query = ("@info(name = 'query1') from inputStream select "
+                + "map:createFromXML(\"<sensor>" +
+                "<commonAttr1>19</commonAttr1>" +
+                "<commonAttr2>11.45</commonAttr2>" +
+                "<commonAttr3>true</commonAttr3>" +
+                "<commonAttr4>ELEMENT_TEXT</commonAttr4>" +
+                "<specAttributesObj>" +
+                "<specAttr1>111</specAttr1>" +
+                "<specAttr2>222</specAttr2>" +
+                "</specAttributesObj>" +
+                "</sensor>\") as hashMap insert into outputStream;" +
+                "from outputStream " +
+                "select map:toXML(hashMap, 'wso2' ,'ibm') as xmlString " +
+                "insert into outputStream2;");
+
+        ExecutionPlanRuntime executionPlanRuntime = siddhiManager.createExecutionPlanRuntime(inStreamDefinition + query);
+
+        InputHandler inputHandler = executionPlanRuntime.getInputHandler("inputStream");
```

Comment:
Below three lines are not needed. createExecutionplanRuntime will throw the exception